### DISCUSSION TITLE
SOC Cases & Safari

### DISCUSSION THREAD

**Discussion by wdhachfeld** [Security-Onion-Solutions/securityonion#7186] [General] [answered]
A couple days ago I upgraded my standalone installation from 2.3.91 to 2.3.100. The upgrade went fine and everything appeared to be working. Today, however, I created a new case for the first time in SOC Cases. And when I go to the Cases panel and click on the View icon for my newly-created case I get the following in (Mac) Safari 15.3:

![Safari](https://user-images.githubusercontent.com/3442933/153653566-ca055d73-0ad8-4fad-8db3-038047c9f686.png)

Where in Chrome 98.0.4758.80 I see:

![Chrome](https://user-images.githubusercontent.com/3442933/153653661-918c9c8b-a6e1-4226-a294-c835ee123049.png)

I'm aware that Safari isn't technically considered a supported browser. This is just an FYI that there is an issue somewhere with SOC Cases on the Mac. In the event you want to look into it, I'm happy to assist by providing any additional information for diagnosing the issue. for now I'll simply stick to using Chrome.

**Comment by dougburks** [comment, marked as answer]
For others who may run across this discussion, here is our official browser recommendation as seen in https://docs.securityonion.net/en/2.3/soc.html:
> We recommend chromium or chromium-based browsers such as Google Chrome. Other browsers may work, but fully updated chromium-based browsers provide the best compatibility.
